packages/cozy-dataproxy-lib/src/search/SearchEngine.ts

```diff
@@ -76,13 +84,32 @@ export class SearchEngine {
     if (!this.client) {
       return
     }
-    // Create search indexes by querying docs, either locally or remotely
-    for (const doctype of SEARCHABLE_DOCTYPES) {
-      const searchIndex = await this.indexDocsForSearch(
-        doctype as keyof typeof SEARCH_SCHEMA
-      )
-      if (searchIndex) {
-        this.searchIndexes[doctype] = searchIndex
+
+    if (this.isLocalSearch) {
```

Comment:
there seems to be no code branch for the remote search anymore. Also those are 2 consecutive `if(this.isLocalSearch)`, should we merge theme?

Reply:
This is definitely not expected, good catch! 
Fixed within 0443ed641964ef3e75e4b264ca5b7dadcb06d156